README.md

```diff
@@ -7,17 +7,17 @@
 [![Smoke Test](https://github.com/factset/go-drill/workflows/SmokeTest/badge.svg)](https://github.com/factset/go-drill/actions)
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 
-go-drill is a highly efficient Pure Go Client and Sql driver for [Apache Drill](https://drill.apache.org). It differs from other clients / drivers by using the
-native Protobuf API to communicate with Drill instead of the REST API. As a result
-this becomes significatly more performant when dealing with large amounts of data.
-The raw bytes that are returned from Drill aren't copied, but are instead
-interpreted and used in place via slices for efficiency.
+**go-drill** is a highly efficient, pure go client and SQL driver for [Apache Drill](https://drill.apache.org).
+It differs from other clients / drivers by using the native Protobuf API to
+communicate with Drill instead of the REST API. The use of Protobufs enables
+zero-copy access to returned data, resulting in greater efficiency.
 
-Currently it only supports either no authentication or authentication via SASL
-gssapi-krb5.
+At the present time, the driver may be used without authentication or with
+authentication via SASL gssapi-krb-5.
```

Comment:
So might as well update this as part of this PR, as of my recent changes, it also supports User/Pass authentication too. :) Just had forgotten to update the readme when I did that.